independent-publisher-connectors/Annature/apiProperties.json

```diff
@@ -0,0 +1,83 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_client_id": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "Annature API Client Id",
+          "displayName": "Annature Api Id",
+          "tooltip": "Specifiy Annature Api Client Id"
+        }
+      },
+      "api_key": {
```

Comment:
@iddycol can you please explain how this connector is using client id and api key?

Reply:
Annature doesn;t uses oAuth and it uses a ID and Key combination for validation. To make this precise, we have update the file with simple word "Annature Id". Please review now  and let us know if you have any queries.